samples/nrf9160/http_application_update/Kconfig

```diff
@@ -16,8 +16,20 @@ config APPLICATION_VERSION
 	int "Application version"
 	default 1
 
+config ENABLE_HTTPS
+	bool "Enable HTTPS download"
+	depends on DOWNLOAD_CLIENT_TLS
+
+config HTTPS_SECURITY_TAG
+	int
+	prompt "HTTPS security tag" if ENABLE_HTTPS
+	default 42
+	help
+	  Set to the security tag which you have stored your HTTPS servers root
+	  CA chain certificate. Leave as -1 to download from regular HTTP
```

Comment:
`Leave as -1 to download from regular HTTP`

The description seems to be inaccurate, `ENABLE_HTTPS` should be used to switch between HTTP/HTTPS. Right now it could be confusing for users.

Reply:
It's a bit confusing I do agree. I'm not certain about what is the best way to expose this option. What I want is for it to be default `-1` when enable HTTPS is off. But if you set it to `-1` manually it will go back to use HTTP. I think the best option would be to add a `range` to the Kconfig.

Reply:
How do I see it:
1. Either give up on `ENABLE_HTTPS` config, and leave secure tag only. Then you can document, that you can set -1 to use regular HTTP,
2. Or leave it as it is, but remove the sentenced I mentioned. In case `ENABLE_HTTPS` is off you don't need to worry about its value anyway as it's not used. And yes, if you want to be sure that noone sets -1 when HTTPS is enabled, you can use `range 0 INT_MAX` (I believe you need to use a numeric value for INT_MAX though).